lutris/gui/gameviews.py

```diff
@@ -137,6 +137,7 @@ class GameView(object):
         "game-selected": (GObject.SIGNAL_RUN_FIRST, None, ()),
         "game-activated": (GObject.SIGNAL_RUN_FIRST, None, ()),
         "game-installed": (GObject.SIGNAL_RUN_FIRST, None, (int,)),
+        "remove-game": (GObject.SIGNAL_RUN_FIRST, None, ())
```

Comment:
Include a trailing comma, like the line above this, since this is the code style of the surrounding code. (See also, [PEP 8: When to use trailing commas](https://www.python.org/dev/peps/pep-0008/#when-to-use-trailing-commas).)